source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -129,6 +240,109 @@ This means that in order to have a VM connected to both ipv4 and ipv6 subnets,
 it needs to connect to 2 different networks (through 2 different logical ports),
 one with an IPv4, other with an IPv6 subnet defined on top.
 
+### Networking API entity provisioning
+
+Below you can find the steps required to have dynamic IPv6 addresses on the
+guest VMs, along with the available methods of provisioning it.
+
+1. create a logical network
+    - oVirt UI
+    - oVirt REST API
+    - manageIQ UI
+    - ovirt-provider-ovn REST API. [REST example](#create-a-logical-network)
+2. add a subnet on top
+    - oVirt UI - *requires changes*
+    - manageIQ UI
+    - ovirt-provider-ovn REST API. [REST example](#create-a-subnet)
+3. provision a router
+    - ovirt-provider-ovn REST API. [REST example](#create-a-router)
+4. attach the subnet to the router
+    - oVirt ovirt-provider-ovn REST API. [REST example](#configure-the-router)
+5. create a VM having the vNIC profile of the network created in step #1
+   **or** assign the vNIC profile of the network created in #1 to an existing
+   VM
+    - oVirt UI
+    - oVirt REST API
+
+#### EL7 Guest VM configuration
+
+For dhcpv6-stateless the guest VM can be configured using *either* of the 3
+configuration options defined in the [Guest config section](#guest-vm-configuration).
+
+It is important to state that if and when the dhclient version supplied in EL7
+stops defaulting to /64 prefixes, the guest VMs have to be configured as
+indicated for [EL8](#el8-guest-vm-configuration).
+
+#### EL8 Guest VM configuration
+
```

Comment:
What do you think about adding a minimal ifcfg file with a short description of the resulting configuration with and without router?

Reply:
I'd prefer to not do that, and advise for a router to be *always* created, since it configures the guest's routes correctly, independent of the scenario.